apps/back-office/src/app/(dashboard)/clients/[id]/diagnosisAndTreatmentPlan/components/TreatmentPlanView.tsx

```
@@ -0,0 +1,477 @@
+
+        // Fetch the treatment plan
+        const planData = await fetchDiagnosisTreatmentPlanById(planId);
+        setPlan(planData);
+
+        // Debug logging
+        console.log("Treatment plan data:", planData);
+        console.log("Survey answers ID:", planData.survey_answers_id);
+        console.log("Survey answers data:", planData.SurveyAnswers);
+
+        // Check if survey data is included in the response
+        if (planData.SurveyAnswers && planData.SurveyAnswers.content) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)